runhouse/main.py

```diff
@@ -631,30 +253,38 @@ def cluster_list(
     # logged out case
     if not rh.configs.token:
         # TODO [SB]: adjust msg formatting (coloring etc)
-        sky_cli_command_formatted = f"{italic_bold_ansi}sky status -r{reset_format}"  # will be printed bold and italic
+        sky_cli_command_formatted = f"{ITALIC_BOLD}sky status -r{RESET_FORMAT}"  # will be printed bold and italic
         console.print(
             f"Listing clusters requires a Runhouse token. Please run  `runhouse login` to get your token, or run {sky_cli_command_formatted} to list locally stored on-demand clusters."
         )
         return
 
     clusters = Cluster.list(show_all=show_all, since=since, status=cluster_status)
 
-    all_clusters = clusters.get("all_clusters", None)
-    running_clusters = clusters.get("running_clusters", None)
+    den_clusters = clusters.get("den_clusters", None)
+    running_clusters = (
+        [
+            den_cluster
+            for den_cluster in den_clusters
+            if den_cluster.get("Status") == "running"
+        ]
+        if den_clusters
+        else None
+    )
     sky_clusters = clusters.get("sky_clusters", None)
 
-    if not all_clusters:
+    if not den_clusters:
         no_clusters_msg = f"No clusters found in Den for {rns_client.username}"
         if show_all or since or cluster_status:
             no_clusters_msg += " that match the provided filters"
         console.print(no_clusters_msg)
         if sky_clusters:
-            print_sky_clusters_msg(len(sky_clusters))
+            console.print(SKY_LIVE_CLUSTERS_MSG)
```

Comment:
this variable is pretty specific to `main.py`, i think we can move it out of constants and put here